themes/default/content/docs/intro/concepts/plans.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Plans"
+meta_desc: Learn about Pulumi update plans and how they can be used.
+menu:
+  intro:
+    parent: concepts
+    weight: 10
+
+aliases:
+  - /updateplans/
+---
+
+{{% notes type="warning" %}}
+Update plans are currently in experimental preview and will only show up in `--help` if the environment variable `PULUMI_EXPERIMENTAL` is set to `true`.
+{{% /notes %}}
+
+The result of previews can be saved to an _update plan_, this can then be used during a later update to constrain the update only to the operations that were planned.
+
+## Generate an update plan
+
+To generate an update plan run `pulumi preview --save-plan=plan.json`. The plan will be written to the `plan.json` file.
+
+## Use an update plan
```

Comment:
We should also:

1. Clearly note the limitations (here or wherever it makes most sense to the author) of plan files, namely that they will not include resources created in an apply. 
2. Give some advice on how to handle this scenario, since unapplied changes in the main branch are not good. (Keep generating plans until there are no additional changes? Would we need to modify the GHA etc. to give a good workflow if people are using plan files?)